### DISCUSSION TITLE
posts/Go/go_mongodb/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#345] [Announcements]
https://www.liwenzhou.com/posts/Go/go_mongodb/ 

 李文周的Blog noSQL mongoDB 芒果 DB go连接MongoDB Go语言 mongoDB驱动

**Comment by woshihaoren4** [comment]
请问官网的mongo-driver是线程安全的吗？

**Comment by Q1mi** [comment]
> @woshihaoren4
> 请问官网的mongo-driver是线程安全的吗？

不明白你的问题

**Comment by yzdzh** [comment]
怎样才能获取原始sql，我想获取原始sql然后插到日志文件中，方便以后排查

**Comment by q1ntq1** [comment]
连接池模式，不需要client.Disconnect 了吗？应该怎样关闭与mongodb的连接呢？

**Comment by powerxu519** [comment]
连接池模式 ConnectToDB函数结束时，context不就关掉了吗，返回的*mongo.Database就为空了，那么这个返回值有什么意义呢
